tests/DeploymentPipelineStepTest.php

```diff
@@ -102,6 +103,21 @@ public function testSnapshotSkipped() {
 	}
 
 	/**
+	 * Test snapshot is skipped upon user request.
+	 */
+	public function testSnapshotSkippedByUser() {
+		$step = $this->getDummyDeployment(false, true);
+		$step->start();
+
+		// Assert not error at startup
+		$this->assertEquals('Started', $step->Status);
+		$this->assertEquals('Deployment', $step->Doing);
+		$this->assertFalse(PipelineTest_MockLog::has_message('TestDeployStep:Snapshot creating snapshot of database'));
+		$snapshot = $step->Pipeline()->PreviousSnapshot();
+		$this->assertFalse($snapshot->exists(), 'No snapshot was created');
```

Comment:
the output will be like:

```
There was 1 failure:

1) DeploymentPipelineStepTest::testSnapshotSkippedByUser
No snapshot was created
Failed asserting that true is false.
```

Would that make sense? You are trying to assert that there no snapshot created. Just thinking about what sort of "standard" we should have in the assert messages, since I try to point out why the test failed. no biggie, just interested in your opinions.